modules/logging.py

```diff
@@ -64,9 +67,32 @@ async def logging_loop(self) -> None:
 
         embed = to_log.__dict__.get("embed") or discord.utils.MISSING
 
+        avatar_url: str | None = core.CONFIG["LOGGING"].get("webhook_avatar_url")
+        avatar_name = "PythonistaBot Logging"
+
+        if not avatar_url and self.user is not None and "runner" in core.CONFIG["LOGGING"]:
+            runner_id: int = core.CONFIG["LOGGING"]["runner"]
+            user = self.user or self.bot.get_user(runner_id)
+
+            if user:
+                avatar_url = user.display_avatar.url
+                avatar_name = f"Logging: Dev: {user.display_name}"
+
+            else:
+                try:
+                    user = await self.bot.fetch_user(
+                        runner_id
+                    )  # dont particularly need to chunk the entire guild for the one user
+                    self.user = user
+                except:
+                    self.user = None  # This will tell us not to attempt again.
+                else:
+                    avatar_url = user.display_avatar.url
+                    avatar_name = f"Logging: Dev: {user.display_name}"
+
         await self.webhook.send(
             message,
-            username="PythonistaBot Logging",
+            username=avatar_name,
```

Comment:
"avatar name" sounds a bit unusual in this case.
I know we changed it to support the new avatar but ideally this would be named something like "actor name" or the like... Nitpick, but yeah